### PR TITLE
add polls for December 30th

### PR DESCRIPTION
**Pull request by 0xldr**
adds polls for:

https://forum.sky.money/t/27-dec-2024-proposed-changes-to-spark-for-upcoming-spell/25760

and

https://forum.sky.money/t/resilience-research-project-application-legal-opinion-on-mica-compliance-for-cex-listings-of-usds-and-dai/25759

### CODE REVIEW

File: governance/polls/Resilience Research Project Application - Legal Opinion on MiCA Compliance for CEX Listings of USDS and DAI - December 30, 2024.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Support Facilitators. This Governance [Poll](https://sky-atlas.powerhouse.io/#A.1.9.1_Operational_Weekly_Cycle-b189fa17-57a9-4d4e-9780-0ce4efd94211%7C0db30308) will be active for three days beginning on Monday, December 30 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to funding this Resilience Research Project Application which seeks to fund a legal opinion on MiCA compliance for CEX listings of USDS and Dai. The project abstract is as follows:
```
Comment (blimpa) [inline governance/polls/Resilience Research Project Application - Legal Opinion on MiCA Compliance for CEX Listings of USDS and DAI - December 30, 2024.md:29]:
```suggestion
The community can vote in this poll to express support or opposition to funding a Resilience Research Project proposal which seeks to produce a legal opinion on MiCA compliance for CEX listings of USDS and Dai. The project abstract is as follows:
```
This language is closer to [what stands in the Atlas](https://sky-atlas.powerhouse.io/A.2.8.1.4_Application_Process/9660eeb7-900e-49ff-9323-e7560b7a6436%7C9e1f609f13e1), I think.

File: governance/polls/Spark Liquidity Layer Base - Onboard Aave V3 Base Market USDC - December 30, 2024.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to onboarding Aave V3 Base market USDC to the Spark Liquidity Layer on Base with the following parameters:
+* `max amount` (maximum amount that can be executed at once) for deposits: **50 million USDC**
```
Comment (blimpa) [inline governance/polls/Spark Liquidity Layer Base - Onboard Aave V3 Base Market USDC - December 30, 2024.md:30]:
```suggestion
* `max amount` (maximum amount that can be executed at once) for deposits: **50 million USDC**.
```

**Review by MakerJanSky — APPROVED** [review]
Nice work, ldr.

File: governance/polls/Spark Liquidity Layer Mainnet - Onboard Ethena USDe and sUSDe Direct Exposure - December 30, 2024.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Stability Facilitators. This Governance [Poll](https://sky-atlas.powerhouse.io/#A.1.9.1_Operational_Weekly_Cycle-b189fa17-57a9-4d4e-9780-0ce4efd94211%7C0db30308) will be active for three days beginning on Monday, December 30 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following changes:
```
Comment (blimpa) [inline governance/polls/Spark Liquidity Layer Mainnet - Onboard Ethena USDe and sUSDe Direct Exposure - December 30, 2024.md:29]:
```suggestion
The community can vote in this poll to express support or opposition to the following  following parameter changes for Spark’s direct Ethena staked USDe exposure:
```

**Review by VoteWizard — APPROVED** [review]

**Comment by MakerJanSky** [comment]
# Poll Review

## Admin I

- [x] Is the poll expected this week?
- [x] Do you understand the question the poll is asking of governance?

## Content

- [x] Does the filename have the .md extension?
- [x] Does the filename contain banned characters? <>:"/\?*|
- [x] Do the titles contain banned characters?  <>:"/\?*
- [x] Are the poll options too long?  (Less than 30 characters as a rough guide) - **Resilience Research Project requires longer options.**
- [x] Do the options have distinct numbers in the preamble?
- [x] Are the start and end dates present and correct?
- [x] Does the vote type match desired vote type for the poll?
- [x] Is the default option correct in metadata?
- [x] Is the preamble ready for submission?
- [x] Is the vote descriptor present and does it match the vote type?
- [x] Are the 'Review', 'Outcomes', and 'Resources' sections present?
- [x] Will the Data/parameters present in the poll be accurate at the time the poll is live?
- [x] Are the dates correct within the text?
- [x] Are all the relevant links provided?
- [x] Are all the links working?
- [x] Are the outcomes clear and comprehensive?
- [x] Do any '$' parameter signifiers remain?
- [x] Does the poll summary accurately reflect the poll content?
- [x] Read the poll text in full and highlight any other potential issues.

## Admin II

- [x] Raise any issues with the poll author via review on GitHub.
- [x] Inform the reviewer when the poll passes review.

**Pull request merged**